TAC/Meetings/2024/2024-06-27/CCC-TAC-Minutes-2024-06-27.md

```diff
@@ -0,0 +1,127 @@
+
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* AMD - David Kaplan+ / Harold Gilkey 
+* Arm - Nathaniel McCallum   / Michael Lu
+* Google - Catherine Zhang+ 
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair)+   / Simon Johnson
+* Meta -  Henry Wang+ /  Kevin Hui 
+* Microsoft - Alec Fernandez+ 
+* Nvidia - Fritz Alder+ 
+* Red Hat - Yash Mankad+  / Ram Pai 
+* TikTok - Mingshen Sun+   / Yao Zhang
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Bokdeuk Jeong (Samsung)
+* Cosmin Aprodu
+* Dayeol Lee (TikTok)
+* Deli Gong
+* Henk Birkholz
+* Manu Foutaine (Hushmesh)
+* Mark Novak (JP Morgan Chase)
+* Mona Vij (Intel)
+* Nicolae Paladi 
+* Ofir Azoulay-Rozanes (Anjuna)
+* Ram Pai (IBM)
+* Vini Jaiswal (TikTok)
+
+* Mike Bursell (LF)
+* Sal Kimmich (LF)
+* Ben Sternthal (BS)
+
+
+## Welcome
+
+Dan Middleton (DM) opened the call at 7:05 am and reviewed the antitrust policy. 
+
+## Agenda 
+
+DM reviewed the agenda for the meeting. 
+
+## Roll Call / Attendance
+
+Since no votes would be occurring in the meeting, attendance was skipped. 
+
+## Welcome New Community Members
+
+* Cosmin Aprodu introduced themselves.
+* Deli Gong introduced themselves.
+
+## Old Business
+
+### Recap CC Summit
+
+In the last meeting the group discussed feedback on the CC summit.
+
+### PQC in CCC
+
+In the last meeting John Manferdelli discussed implementing PQC
+
+## Announcements
+
+Sal Kimmich (SK) stated that the mentorship program is now open and that mentorships can begin at anytime. SK asked interested folks to reach out to Sal or Yash.
+
+Mike Bursell (MB) expressed his interest in meeting with companies or colleagues in Singapore or Seoul at the upcoming PET Summit Singapore. Please reach out to him if you are interested.
+
+## TAC Objectives / TAC Goals: Ecosystem
+
+DM reviewed the TAC’s 2024 objectives and the three focus areas: Projects, Ecosystem, and Community. Alec Fernandez (AF) reviewed the 2024 objective for the GRC SIG. AF then discussed the three proposed Key Results. Discussion ensued.  Mark Novak (MN) suggested collaborating with other SIGs and groups, such as outreach, to help complete KRs. 
+
+MN shared the SIGs discussion of creating expectation patterns and shared a drive folder of [documents](https://drive.google.com/drive/u/0/folders/1n_wmIWPm3qwll78iRRIAzSGNXB_VdGwD). MN reviewed the first pattern they are developing: [Confidential Payload Governance](https://docs.google.com/document/d/1jzoEWmPCeAWV_g6kcJ3pzX9_UdljraM_2qVnwW8sT_s/edit). Discussion ensued. DM encouraged folks to join the GRC meeting and to leave feedback in the document. 
+
+## Announcements
+
+DM stated that the CCC is responding to three NIST documents. AF was unsure if these were the documents that were responded to.
```

Comment:
@riaankleinhans can you clarify about the NIST documents in the slides. Unclear if these are in progress and who has ownership or suggested them.

Reply:
It came in the TAC mailing list, sugesting the TAC could comment. link in the slides
https://lists.confidentialcomputing.io/g/tac/message/1327